HockeySimulator.cbproj.local

```diff
@@ -4,6 +4,7 @@
     <VirtualFolder ID="{04B8677E-6874-4B0C-B524-2D8AF3F1C749}"/>
     <VirtualFolder ID="{AAE5AB75-5277-44D1-97C8-0519464A5CCD}"/>
```

Comment:
You need to delete this file locally, commit and push. This is local file, we don't need to store it in Github(you can see that it contains relative paths that you are using on your computer.